### PR TITLE
Release/v4.0.7

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- #369 

# Review By (Date)
- EOD August 1

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code diff

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@forrestglick Release has been prepped - please let us know when it's ready to be push to production.

**Comment by forrestglick** [comment]
@yvonnetangsu Thanks for the quick work! I reviewed on Dev and noted one small issue: on other entries, including an adjacent entry, it says: "To Be Named". On this new entry it says: "To be Named" – noting the lowercase letter "b". Would it be possible to update the case to match the other entries? With this change, I approve the push to Prod. Thank you!
The page where you can see the entry: https://dev--adapt-giving.netlify.app/endowed-positions/school-of-humanities-and-sciences/?page=3

File: src/fixtures/endowedPositions.json [hunk 1]
```diff
@@ -2404,7 +2404,7 @@
     "Key": "H&SKAAYL5775282",
     "SUBCATEGORY": "SCHOOL OF HUMANITIES AND SCIENCES",
     "POSITION": "The Denning Family Professorship in the Arts",
-    "CURRENT HOLDER": "Louise Glück",
+    "CURRENT HOLDER": "To be Named",
```
Comment (yvonnetangsu) [inline src/fixtures/endowedPositions.json:2407]:
```suggestion
    "CURRENT HOLDER": "To Be Named",
```

**Review by sherakama — APPROVED** [review]

**Comment by yvonnetangsu** [comment]
Thanks for the review @sherakama!

@forrestglick Updated to "To Be Named" to match the other entries.

**Comment by yvonnetangsu** [comment]
@forrestglick Changes are live!
Screenshot 2023-08-01 at 3 37 53 PM

**Comment by forrestglick** [comment]
Awesome! Thanks to @yvonnetangsu and @sherakama for the quick work. I'll let the client know it's live.

**Pull request merged**